tock/hours/views.py

```diff
@@ -313,6 +315,64 @@ def form_valid(self, form):
     def get_success_url(self):
         return reverse("ListReportingPeriods")
 
+def display_add_hours_error_message(request):
+    error_msg = "Oops. That command was not correct and no time was added to your timecard. Try again by entering a URL with this format: tock.gov/addHours?project=231&hours=1"
+    messages.add_message(
+        request,
+        messages.ERROR,
+        error_msg
+    )
+
+
+def add_hours_view(request):
+    project_id = request.GET['project']
+    hours = Decimal(request.GET['hours'])
+    r = ReportingPeriod.objects.latest()
+
+    try:
```

Comment:
@elg0nz Also please include a test case where the user submits an invalid value for the `hours=` param.